lua/lazyvim/plugins/editor.lua

```diff
@@ -24,6 +24,26 @@ return {
         desc = "Explorer NeoTree (cwd)",
       },
       { "<leader>e", "<leader>fe", desc = "Explorer NeoTree (Root Dir)", remap = true },
+      {
+        "<Leader>B",
+        function()
+          local function buffer_dir()
+            return vim.fn.expand("%:p:h")
+          end
+          require("neo-tree.command").execute({ toggle = true, dir = buffer_dir() })
+        end,
+        desc = "Explorer NeoTree (Buffer Dir)",
+      },
```

Comment:
```suggestion
     { "<leader>B", "<leader>fB", desc = "Explorer NeoTree (Buffer Dir)", remap = true },
```
You could just remap it instead of defining it twice? Like in line 26.

Reply:
Thank you for the suggestion i am not that familiar with the lazy.nvim methods.
I have changed it.